apps/internal/jarvis/next.config.js

```diff
@@ -0,0 +1,15 @@
+// eslint-disable-next-line @typescript-eslint/no-var-requires
+const withNx = require('@nrwl/next/plugins/with-nx');
```

Comment:
tell me about this part plz.

Reply:
This is auto generated by NX, it updates webpack config, I couldn't find any info about it online, but you can enter the `with-nx.js` file and see the comments explaining what does it do.